app/controllers/tile_metadata_controller.rb

```diff
@@ -1,19 +1,18 @@
 # frozen_string_literal: true
 class TileMetadataController < ApplicationController
   attr_reader :resource
+  rescue_from MosaicService::Error, with: :not_found
 
   def metadata
     @resource = find_resource(params[:id])
-    if resource.is_a?(RasterResource) && resource.decorate.raster_set?
+    if resource.is_a?(RasterResource) || resource.is_a?(ScannedMap)
```

Comment:
Let's add a code comment here saying why we purposefully include things we know will not have a mosaic service.